arangod/RocksDBEngine/RocksDBBuilderIndex.cpp

```diff
@@ -720,15 +338,12 @@ static arangodb::Result fillIndex(
                               threadBatchSize, dbOptions, idxPath);
 #ifdef USE_SST_INGESTION
       if (res.ok()) {
-        for (auto const& fileName : TRI_FullTreeDirectory(idxPath.data())) {
-          TRI_UnlinkFile(
-              basics::FileUtils::buildFilename(idxPath.data(), fileName.data())
-                  .data());
-        }
+        cleanUpTempRocksDBDir(idxPath.data());
```

Comment:
We should pass the proper std::string here and not a `char const*`.
```suggestion
        cleanUpTempRocksDBDir(idxPath);
```